spec/administrate/views/fields/has_one/_show_spec.rb

```diff
@@ -43,15 +51,54 @@
       expect(rendered.strip).to include(link)
     end
 
-    def nested_show
-      instance_double(
+    it "renders nested attribute relationships" do
+      template.extend Administrate::ApplicationHelper
+
+      product = create(:product)
+      page = create(:page, product: product)
+
+      nested_has_many = instance_double(
+        "Administrate::Field::HasMany",
+        associated_collection: [page],
+        attribute: :page,
+        data: [page],
+        resources: [page],
+        html_class: "has-many",
+        name: "Page",
+        to_partial_path: "fields/has_many/index",
+        order_from_params: {},
+      )
```

Comment:
Similarly, I think we can provide a real `Administrate::Field::HasMany`. Here's an example:

https://github.com/thoughtbot/administrate/blob/64aaa98a79b3108fb1442025eb818d3aea8e7b3d/spec/lib/fields/has_many_spec.rb#L22-L27

Reply:
Scratching my head on this one:

```
Failure/Error: <%= render(
     
     ActionView::Template::Error:
       Missing partial /_collection, fields/has_one/_collection with {:locale=>[:en], :formats=>[:html, :text, :js, :css, :ics, :csv, :vcf, :vtt, :png, :jpeg, :gif, :bmp, :tiff, :svg, :mpeg, :mp3, :ogg, :m4a, :webm, :mp4, :otf, :ttf, :woff, :woff2, :xml, :rss, :atom, :yaml, :multipart_form, :url_encoded_form, :json, :pdf, :zip, :gzip], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby]}. Searched in:
         * "/Users/brianfield/code/buildrtech/administrate/spec/example_app/app/views"
         * "/Users/brianfield/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/administrate-field-image-1.1.0/app/views"
         * "/Users/brianfield/code/buildrtech/administrate/app/views"
         * "/Users/brianfield/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/kaminari-core-1.2.1/app/views"
```

```diff
diff --git a/spec/administrate/views/fields/has_one/_show_spec.rb b/spec/administrate/views/fields/has_one/_show_spec.rb
index 6ca4309..31d75ed 100644
--- a/spec/administrate/views/fields/has_one/_show_spec.rb
+++ b/spec/administrate/views/fields/has_one/_show_spec.rb
@@ -1,5 +1,6 @@
 require "rails_helper"
 require "administrate/field/has_one"
+require "administrate/field/has_many"
 
 describe "fields/has_one/_show", type: :view do
   context "without an associated record" do
@@ -25,9 +26,7 @@ describe "fields/has_one/_show", type: :view do
       product_path = polymorphic_path([:admin, product])
       nested_show = instance_double(
         "Administrate::Page::Show",
-        resource: double(
-          class: ProductMetaTag,
-        ),
+        resource: create(:product_meta_tag),
         attributes: [],
         resource_name: "Product Tag",
       )
@@ -52,28 +51,22 @@ describe "fields/has_one/_show", type: :view do
     end
 
     it "renders nested attribute relationships" do
-      template.extend Administrate::ApplicationHelper
+      view.extend Administrate::ApplicationHelper
 
       product = create(:product)
       page = create(:page, product: product)
+      product_meta_tag = create(:product_meta_tag, product: product)
 
-      nested_has_many = instance_double(
-        "Administrate::Field::HasMany",
-        associated_collection: [page],
-        attribute: :page,
-        data: [page],
-        resources: [page],
-        html_class: "has-many",
-        name: "Page",
-        to_partial_path: "fields/has_many/index",
-        order_from_params: {},
+      nested_has_many = Administrate::Field::HasMany.new(
+        :pages,
+        Page.all,
+        :show,
+        resource: product,
       )
 
       nested_show = instance_double(
         "Administrate::Page::Show",
-        resource: double(
-          class: ProductMetaTag,
-        ),
+        resource: product_meta_tag,
         attributes: [nested_has_many],
         resource_name: "Product Tag",
       )
```

Something simple I'm missing?